static/index.html

```diff
@@ -76,6 +83,17 @@ <h2>PayJoin here to open these channels</h2>
 				table.appendChild(div.firstChild);
 				table.appendChild(div.lastChild)
 			}
+
+			function remove_channel(e) {
+				const table = document.getElementById("channels");
+				if (table.children.length <= 2) {
+					return
+				}
+				// Remove last two inputs
+				table.removeChild(table.children[table.children.length - 1])
+				table.removeChild(table.children[table.children.length - 2])
```

Comment:
maybe magic number are a sign the rows belong in 1 element so we can do `channels.removeChild(channels.lastChild);`

Reply:
your original approach was the right one after all